tensorflow/compiler/tf2tensorrt/utils/py_utils.cc

```diff
@@ -35,10 +35,7 @@ bool IsGoogleTensorRTEnabled() {
 #else   // TF_USE_TENSORRT_STATIC
   auto handle_or = se::internal::DsoLoader::TryDlopenTensorRTLibraries();
   if (!handle_or.ok()) {
-    LOG_WARNING_WITH_PREFIX
-        << "Cannot dlopen some TensorRT libraries. If you would like "
-           "to use Nvidia GPU with TensorRT, please make sure the "
-           "missing libraries mentioned above are installed properly.";
+    LOG_WARNING_WITH_PREFIX << "TensorRT is not installed.";
```

Comment:
Maybe users just installed it to the wrong path. How about "Could not find TensorRT." ?

@reedwm What do you think?

Reply:
"Could not find TensorRT." SGTM.

I think ideally, we would only give this message if CUDA drivers were installed. But I'm not sure how difficult that is, and regardless no need to do this in this PR.

Reply:
Thanks Penporn and Reed, I will make the change.